google/cloud/automl/unit-tests/cloud_automl_vision_model_tests.yaml

```diff
@@ -0,0 +1,70 @@
+test:
+  suites:
+  - name: Cloud AutoML Vision Model API IT
+    setup:
+      # Read Project ID from Environment Variable
+    - code: project_id="java-docs-samples-testing"
```

Comment:
Read model & project ID from environment variables (see other comment)

Reply:
But for unit testing, are we supposed to use this project?